coalib/settings/FunctionMetadata.py

```diff
@@ -101,6 +106,7 @@ def add_alias(self, original, alias):
         :param alias:     The name of the alias for the original.
         :raises KeyError: If the new setting doesn't exist in the metadata.
         """
+        self.deprecated_params.append(alias)
```

Comment:
also we don't want to have deprecated paras in the optional_params und non_optional_params list do we?